spring-cloud-gcp-docs/src/main/asciidoc/spring-stream.adoc

```diff
@@ -57,10 +57,10 @@ spring.cloud.stream.gcp.pubsub.bindings.events.producer.auto-create-resources=tr
 ==== Consumer Destination Configuration
 
 If automatic resource creation is turned ON and the subscription and/or the topic do not exist for a consumer, a subscription and potentially a topic will be created.
-The topic name will be the same as the destination name, and the subscription name will be the same as the consumer group name.
+The topic name will be the same as the destination name, and the subscription name will be the destination name followed by the consumer group name.
 If the consumer group is not specified, an anonymous one will be created with the name `anonymous.<destinationName>.<randomUUID>`.
 
-For example, for the following configuration, a topic named `myEvents` and a subscription called `counsumerGroup1` would be created.
+For example, for the following configuration, a topic named `myEvents` and a subscription called `myEvents.counsumerGroup1` would be created.
```

Comment:
I think we need to add some IMPORTANT note that created subscriptions in advance should follow this pattern otherwise a new one is going to be created to confuse end-user.
Or more over when we configure an `auto-create-resources` to `false` and end-user is going to use an existing subscription he/she is going to be surprised that binding doesn't work because there is no `[name].[group]` subscription.

Reply:
Good point. Done.